composer.json

```diff
@@ -66,19 +66,19 @@
         "google/cloud-bigquery": "0.2.3",
         "google/cloud-core": "1.8.1",
         "google/cloud-datastore": "1.0.1",
-        "google/cloud-error-reporting": "0.4.2",
+        "google/cloud-error-reporting": "0.4.3",
         "google/cloud-logging": "1.3.2",
-        "google/cloud-monitoring": "0.4.2",
-        "google/cloud-language": "0.4.1",
+        "google/cloud-monitoring": "0.4.3",
+        "google/cloud-language": "0.5.0",
         "google/cloud-pubsub": "0.6.2",
-        "google/cloud-spanner": "0.5.0",
+        "google/cloud-spanner": "0.5.1",
         "google/cloud-speech": "0.6.2",
         "google/cloud-storage": "1.1.4",
         "google/cloud-trace": "0.3.2",
         "google/cloud-translate": "1.0.1",
-        "google/cloud-videointelligence": "0.3.2",
+        "google/cloud-videointelligence": "0.3.3",
         "google/cloud-vision": "0.4.1",
-        "google/cloud-dlp": "0.1.1"
+        "google/cloud-dlp": "0.2.0"
```

Comment:
Could this be a patch? It looks like the only changes were adding new documentation and bumping gax/proto-client.